src/modules/mavlink/module.yaml

```diff
@@ -44,6 +44,8 @@ parameters:
                 #6: Iridium # as the user does not need to configure this, hide it from the UI
                 7: Minimal
                 8: External Vision
+                #9: External Vision Minimal # hidden
```

Comment:
Why hide this?

Reply:
I don't remember, I think that's how it was.